packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx

```diff
@@ -75,7 +75,19 @@ export function ConnectionItem(props: ConnectionItemProps) {
               line-height: 16px;
             `}
           >
-            {props.item.title}
+            <span
+              css={`
+                font-size: 10px;
+                background: rgba(255, 255, 255, 0.25);
+                opacity: 0.85;
+                padding: 1px 2px;
+                margin-right: 4px;
+                border-radius: 4px;
+              `}
+            >
+              {getKindName(props.item.kind)}
+            </span>
+            <span>{props.item.title}</span>
```

Comment:
Could you add `vertical-align: middle` to the span with connection title? It should align those two spans that have different font sizes.

Reply:
Done 👍